### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mkamol21**
Testing pull request

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,3 +73,5 @@ gulp
 ### Deleting a category
 
 - Attemping to delete a category will activate a dialog to confirm the action.
+
```
Comment (mkamol21) [inline README.md:76]: I like line 76

Comment (basham) [inline README.md:76]: Well, I don't.

File: README.md [hunk 2]
```diff
@@ -73,3 +73,5 @@ gulp
...
+Kamol was here
```
Comment (basham) [inline README.md:77]: This is the line is what you added.

**Comment by basham** [comment]
Let's close this, and try to pull it into the `help` branch, please.

**Pull request closed without merging**